runelite-client/src/main/java/net/runelite/client/plugins/cluescrolls/clues/EmoteClue.java

```diff
@@ -93,6 +94,12 @@ public class EmoteClue extends ClueScroll implements LocationClueScroll
 		ItemVariationMapping.getVariations(PHARAOHS_SCEPTRE).stream()
 			.map(ItemRequirements::item)
 			.toArray(SingleItemRequirement[]::new));
+	private static final AnyRequirementCollection CRYSTAL_BOW_OR_BOW_OF_FAERDHINEN = any("Crystal Bow or Bow of Faerdhinen",
+		Stream.of(ItemVariationMapping.getVariations(BOW_OF_FAERDHINEN_INACTIVE).stream(), ItemVariationMapping.getVariations(NEW_CRYSTAL_BOW).stream())
```

Comment:
`ItemVariationMapping.getVariations(NEW_CRYSTAL_BOW)` isn't helpful here because it will include [the inactive crystal bow](https://chisel.weirdgloop.org/moid/item_id.html?23985#23970-24000) which does not work for this clue. I recommend using `Stream.of(CRYSTAL_BOW, CRYSTAL_BOW_24123)` instead.

Additionally, to make the formatting clearer here since we're streaming several inputs, I'd like to see this formatted as follows:

```java
private static final AnyRequirementCollection CRYSTAL_BOW_OR_BOW_OF_FAERDHINEN = any("Crystal Bow or Bow of Faerdhinen",
	Stream.of(
		ItemVariationMapping.getVariations(BOW_OF_FAERDHINEN_INACTIVE).stream(),
		Stream.of(CRYSTAL_BOW, CRYSTAL_BOW_24123))
		.reduce(Stream::concat)
		.orElseGet(Stream::empty)
		.map(ItemRequirements::item)
		.toArray(SingleItemRequirement[]::new));

Reply:
Would it make more sense to keep using the ItemVariationMapping for the Crystal bow and remove the inactive one in the stream? Meets your aim then of using the mapper to reduce overhead going forwards.

Do we know for sure if the inactive bowfa version works, otherwise that’ll need the same treatment? Ticket said all versions so assuming so but if the inactive crystal bow doesnt…?

Reply:
[Based on this comment](https://github.com/runelite/runelite/issues/16144#issuecomment-1383226914), I'd think the inactive bowfa is also not usable for this clue. I'd recommend filtering the inactive form like is done in 835c2d2918652e5993b2ca5b2ffc08bf7aaaa23f, but leaving the crystal bow detection the same as it was before, since we know that works as-is and doesn't seem to need any variation mapping.